### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by jpalazz2**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1 +1,155 @@
...
+###Style Guide
+This section has information on how the code should be styled.
+
+####General
+* 79 characters or less per line
+* Docstrings and comments should be 72 characters or less
+* Use four spaces rather than tab characters
+* Surround function and class definitions with two blank lines
+* imports should be on separate lines, unless importing multiple items from one module
+* Imports should be in the following order:
```
Comment (hallfox) [inline CONTRIBUTING.md:82]: Whoops I need to fix this, I've been doing third party, local, standard

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1 +1,155 @@
...
+```
+foo = long_function_name(var_one, var_two
+                         var_three, var_four)
+```
+If the function name is very long and the parameters can't fit on the same line use the following style:
+```
+def really_long_function_name_no_params_fit(
+        var_one, var_two
+        var_three, var_four):
+    print("Hello world!")
```
Comment (hallfox) [inline CONTRIBUTING.md]: Might want to make the " into ' just to be less confusing

**Comment by hallfox** [comment]
I'm convinced, again waiting on @alanplotko

**Comment by alanplotko** [comment]
Beautiful work! I love the pet peeves section and explanation of labels. Having some beginner-labeled issues will be great for new contributors. Looks good on my end!

**Pull request merged**